Desktop/components/JASP/Widgets/VariablesWindow.qml

```diff
@@ -184,7 +184,7 @@ FocusScope
 						value:				columnModel.columnTitle
 						onValueChanged:		if(columnModel.columnTitle !== value) columnModel.columnTitle = value
 						undoModel:			columnModel
-						controlLabel.width:	Math.max(columnTitleVariablesWindow.controlLabel.implicitWidth, descriptionLabel.implicitWidth)
+						controlLabel.width:	Math.max(columnTitleVariablesWindow.label.length, descriptionLabel.implicitWidth)
```

Comment:
Hmm, this is using the length of the text in the label itself, while `implicitWidth` is supposed to be the actual width.

Im not saying we shouldnt use a different variable here or handle the change of the label after a languagechange.
Ill have a look as well

Reply:
Then I will dorp that commit from this PR.